spec/unit/sources_manager_spec.rb

```diff
@@ -181,6 +193,8 @@ module Pod
         it 'redirect' do
           WebMock.stub_request(:get, 'http://some_host.com/something/CocoaPods-version.yml').
             to_return(:status => 301, :body => '', :headers => { 'Location' => ['http://some_host.com'] })
+          WebMock.stub_request(:get, 'http://some_host.com').
+            to_return(:status => 200, :body => '', :headers => {})
```

Comment:
I think this is needed, or `open-uri` will trying to request to the Internet,
can proof this by `WebMock.disable_net_connect!`

Reply:
Agreed.